### PR TITLE
Added event handlers

### PR DESCRIPTION
**Pull request by Mithi-code**
### Added class component for event handlers

- Used class-based component.
- Class states

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,41 +1,139 @@
...
+  }
+
+  handleClick(e) {
+    if (!e.target.name) return;
+    const { next, total, operation } = calculate(this.state, e.target.name);
+    this.setState({ next, total, operation });
+  }
+
   render() {
+    /* eslint-disable */
```
Comment (Lameck1) [inline src/components/Calculator.js]: - [ ] It is advisable that when we disable eslint it should only be a single rule and not eslint itself. We are also expected to state why we did that in a note to the reviewer. Please re-enable eslint.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,41 +1,139 @@
 import React, { Component } from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
```
Comment (Lameck1) [inline src/components/Calculator.js]: - [ ] Issue raised by this disabled rule can be solved and it seems you have already solved it. The solution is adding a `constructor` to the Calculator class. Please re-enable this rule and see if the warning persists. I believe it shouldn't.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @Mithi-code, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### A NOTE
Linters are part and parcel of general code quality standards/conventions that we students are being taught to abide by in our codebases. As such, we are not allowed to completely disable them. It is only recommended to disable specific troublesome rules that one figure out and explain that decision in your PR. Still, the Code Reviewer will need to determine whether the disabled rule has a solution or it really deserved to be disabled.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada:  :champagne:  :confetti_ball: 
Hi @Mithi-code       :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
- Your Design looks awesome keep it up. ✔️  
- No Linter errors :heavy_check_mark: 
- Your README file looks very professional :+1: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**